community/ibm-rstudio-dev/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "rstudio.fullname" . }}
+  labels:
+    app: {{ template "rstudio.name" . }}
+    chart: {{ template "rstudio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "rstudio.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "rstudio.name" . }}
+        release: {{ .Release.Name }}
+        chart: {{ template "rstudio.chart" . }}
+        heritage: {{ .Release.Service }}
+      annotations:
+        productName: rstudio
+        productID: rstudio_{{ .Chart.Version }}
+        productVersion: {{ .Chart.Version }}
+      {{- if .Values.annotations }}
```

Comment:
`annotations` was removed from values.yaml.  If still needed, please re-add to `values.yaml` and `values-metadata.yaml`